packages/@ama-sdk/core/src/fwk/api.helpers.ts

```diff
@@ -52,6 +217,22 @@ export function filterUndefinedValues(object?: { [key: string]: string | undefin
     : {};
 }
 
+/**
+ * Returns a filtered json object, removing all the undefined query parameters
+ * @param object JSON object to filter
+ * @returns an object without undefined values
+ */
+export function filterUndefinedQueryParams(object?: { [key: string]: ParamSerialization | undefined }): { [key: string]: ParamSerialization } {
+  return object
+    ? Object.keys(object)
+      .filter((objectKey) => !!object[objectKey])
+      .reduce<{ [key: string]: ParamSerialization }>((acc, objectKey) => {
+        acc[objectKey] = object[objectKey] as ParamSerialization;
+        return acc;
+      }, {})
```

Comment:
```suggestion
    ? Object.fromEntries(Object.entries(object).filter(([_key, value]) => typeof value !== 'undefined'))
```

Reply:
(you can even remove `_key`:
```suggestion
    ? Object.fromEntries(Object.entries(object).filter(([, value]) => typeof value !== 'undefined'))
```
)